packages/cli/test/integration/generators/controller.integration.js

```diff
@@ -156,16 +155,22 @@ describe('lb4 controller', () => {
         restCLIInput,
       );
 
-      return expect(
-        testUtils
-          .executeGenerator(generator)
-          .inDir(SANDBOX_PATH, () =>
-            testUtils.givenLBProject(SANDBOX_PATH, {
-              includeDummyRepository: true,
-            }),
-          )
-          .withPrompts(noModelInput),
-      ).to.be.rejectedWith(/No models found in /);
+      return testUtils
+        .executeGenerator(generator)
+        .inDir(SANDBOX_PATH, () =>
+          testUtils.givenLBProject(SANDBOX_PATH, {
+            includeDummyRepository: true,
+          }),
+        )
+        .withPrompts(noModelInput)
+        .then(
+          () => {
+            throw new Error('Generator should have failed.');
+          },
+          err => {
+            expect(err).to.match(/No models found in /);
```

Comment:
As I understand `expect().to.be.rejectedWith(/regexp/)`, it works exactly the same way as the new code which you wrote here using `.then(onSuccess, onFailure)`. I personally prefer `rejectedWith` as it's less code to maintain and makes it more difficult to introduce an error.

What are the reasoning behind this change? Can we revert it please?